romancal/datamodels/schemas/reference_files/wfi_img_photom.schema.yaml

```diff
@@ -0,0 +1,18 @@
+%YAML 1.1
+---
+$schema: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/wfi_img_photom.schema"
+title: WFI imaging photometric flux conversion data model
+allOf:
+- $ref: referencefile.schema
+- type: object
+  properties:
+    phot_table:
+      title: Photometric flux conversion factors table
+      datatype:
+      - name: optical_element
+        datatype: [ascii, 12]
+      - name: photmjsr
+        datatype: float32
+      - name: uncertainty
+        datatype: float32
```

Comment:
If this is flux per some area do we need an area for each pixel? I assume that the JWST value is an average? or is all this information in the area ref file?

Reply:
From the photom reference file, the JWST scalar conversion factor is copied to 'photmjsr', which is the nominal surface brightness in MJy/steradian. The area of each pixel (in steradians) is found by multiplying PIXAR_SR (the average area of the pixels, in steradians), by that pixel's area from the pixel area reference file, which is the dimensionless relative area of the pixel.